google-datacatalog-qlik-connector/src/google/datacatalog_connectors/qlik/scrape/engine_api_sheets_helper.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/python
+#
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+import logging
+
+from google.datacatalog_connectors.qlik.scrape import \
+    base_engine_api_helper, websocket_responses_manager
+
+
+class EngineAPISheetsHelper(base_engine_api_helper.BaseEngineAPIHelper):
+    # Keys to be used in the pending reponse ids lists.
+    __DOC_INTERFACES = 'doc_interfaces'
+    __SHEETS = 'sheets'
+
+    def get_sheets(self, app_id, max_seconds_to_wait=60):
+        try:
+            return self._run_until_complete(self.__get_sheets(app_id),
+                                            max_seconds_to_wait)
+        except Exception as e:
+            logging.warning(e)
```

Comment:
isn't better to use:
`
logger.warning("error on get_sheets:",  exc_info=True)
`

From Python logging docs:
```
If exc_info does not evaluate as false, it causes exception information to be added to the logging message. If an exception tuple (in the format returned by sys.exc_info()) or an exception instance is provided, it is used; otherwise, sys.exc_info() is called to get the exception information.
```

that way it will output the exception same as:
`
logging.exception
`

Reply:
Cool, tks!